packages/contracts/package.json

```diff
@@ -0,0 +1,126 @@
+{
+  "name": "@gnosis.pm/dex-contracts",
+  "version": "0.3.2",
+  "description": "Contracts for dFusion multi-token batch auction exchange",
+  "main": "build/common/src/index.js",
+  "types": "build/common/src/index.d.ts",
+  "ts:main": "src/index.ts",
+  "module": "build/esm/src/index.js",
+  "jsnext:main": "build/esm/src/index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "build": "truffle build && yarn run networks-reset && yarn run ts",
+    "test": "truffle test {build/common/,}test/*js && yarn run test-ts",
+    "test-stablex": "yarn run test-js test/stablex/*",
+    "test-js": "truffle test",
+    "test-ts": "mocha -r ts-node/register 'test/models/**/*spec.ts'",
+    "test-streamed-orderbook": "TEST_STREAMED_ORDERBOOK_E2E=1 mocha -r ts-node/register test/models/streamed/index.spec.ts",
+    "coverage": "truffle run coverage --network coverage --file \"{build/common/,}test/*js\"",
+    "networks-extract": "CONF_FILE=$(pwd)'/migration_conf.js'  node node_modules/@gnosis.pm/util-contracts/src/extract_network_info.js",
+    "networks-inject": "CONF_FILE=$(pwd)'/migration_conf.js'  node node_modules/@gnosis.pm/util-contracts/src/inject_network_info.js",
+    "networks-reset": "mkdir -p build/contracts && truffle networks --clean && yarn run networks-inject",
+    "verify-stablex": "truffle run verify BatchExchange",
+    "lint": "yarn run lint:solidity && yarn run lint:ts",
+    "lint:solidity": "solhint \"contracts/**/*.sol\" && solium -d contracts/",
```

Comment:
Unfortunately, doing a `git mv` doesn't make the diff better :(. This is the only line that changed from before:
```diff
diff --git a/package.json b/packages/contracts/package.json
index b768d7c..7aea406 100644
--- a/package.json
+++ b/packages/contracts/package.json
@@ -23,7 +23,7 @@
     "networks-reset": "mkdir -p build/contracts && truffle networks --clean && yarn run networks-inject",
     "verify-stablex": "truffle run verify BatchExchange",
     "lint": "yarn run lint:solidity && yarn run lint:ts",
-    "lint:solidity": "solhint \"contracts/**/*.sol\"",
+    "lint:solidity": "solhint \"contracts/**/*.sol\" && solium -d contracts/",
     "lint:ts": "eslint . --ext .js,.ts",
     "prepack": "yarn run build",
     "prettier": "yarn run prettier:solidity && yarn run prettier:ts",
```